seed_files/scripts/get-tgencounterstat.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/bash
```

Comment:
This script should be removed and a new one created for all example data.

Reply:
Rather than remove the scripts and load one giant script I would prefer a script that calls each in turn